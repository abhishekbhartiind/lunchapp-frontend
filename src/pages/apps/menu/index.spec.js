import React from 'react'
import renderer from 'react-test-renderer'
import { MockedProvider } from 'react-apollo/test-utils'
import gql from 'graphql-tag'
import Menu from './index'

// eslint-disable-next-line no-undef
it('menu correctly', () => {
	const GET_MENUS_BY_SITE = gql`
		query($siteId: String!) {
			menusBySite(siteId: $siteId) {
				_id
				name
			}
		}
	`
	const mocks = [
		{
			request: {
				query: GET_MENUS_BY_SITE,
				variables: { siteId: 'fcfd3a00-a22e-11e9-9e72-bb7f31d38655' }
			},
			result: {
				data: {
					menusBySite: [
						{
							_id: '357f4ca0-a861-11e9-ba38-9f9f09ef187f',
							name: 'Menu Deli 2/4/6'
						},
						{
							_id: '2f7c7b70-add4-11e9-bf32-7f74609da087',
							name: 'Menu Deli HH 3/5/7'
						},
						{
							_id: '1c001ec0-add5-11e9-bf32-7f74609da087',
							name: 'Menu Deli 2/4/6'
						}
					]
				}
			}
		}
	]
	const menu = renderer
		.create(
			<MockedProvider addTypename={false} mocks={mocks}>
				<Menu />
			</MockedProvider>
		)
		.toJSON()
	// eslint-disable-next-line no-undef
	expect(menu).toMatchSnapshot()
})
