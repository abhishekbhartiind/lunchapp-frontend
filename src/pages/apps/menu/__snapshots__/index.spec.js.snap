// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`menu correctly 1`] = `
<div
  className="menu"
>
  <h1
    style={
      Object {
        "color": "#fff",
        "margin": "16px 24px",
      }
    }
  >
    Quản lý menu
  </h1>
  <div
    className="ant-row menu-list"
  >
    <div
      className="ant-skeleton ant-skeleton-active"
    >
      <div
        className="ant-skeleton-content"
      >
        <h3
          className="ant-skeleton-title"
          style={
            Object {
              "width": "38%",
            }
          }
        />
        <ul
          className="ant-skeleton-paragraph"
        >
          <li
            style={
              Object {
                "width": undefined,
              }
            }
          />
          <li
            style={
              Object {
                "width": undefined,
              }
            }
          />
          <li
            style={
              Object {
                "width": "61%",
              }
            }
          />
        </ul>
      </div>
    </div>
  </div>
</div>
`;
